torch/_dynamo/convert_frame.py

```diff
@@ -437,7 +439,7 @@ def transform(instructions, code_options):
         CleanupManager.instance[out_code] = output.cleanups
         check_fn = CheckFunctionManager(
             output,
-            output.guards,
+            output.tracing_context.guards_context.dynamo_guards,
```

Comment:
so you're not going to alias guards to the context anymore? (or even just define a `@property`)

Reply:
you did update guards, why did you update this

Reply:
So actually, I wanted to kill it here altogether, lol, one moment.